src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -850,6 +853,10 @@ export class ExcludeSettingWidget extends ListSettingWidget {
 		return ['setting-list-exclude-widget'];
 	}
 
+	protected override addDragAndDrop(rowElement: HTMLElement, item: IListDataItem, idx: number) {
```

Comment:
Actually shouldn't this only be implemented in `ListSettingWidget`? Does this mean that basic object settings can do DND too?

Reply:
It's only been implemented in the `ListSettingWidget`. The AbstractListWidget doesn't have the `addDragAndDrop` method.